src/integrationTest/java/uk/gov/hmcts/reform/sscs/bulkscancore/ccd/CaseDataHelperTest.java

```diff
@@ -0,0 +1,246 @@
+package uk.gov.hmcts.reform.sscs.bulkscancore.ccd;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static com.google.common.io.Resources.getResource;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.ThrowableAssert.catchThrowable;
+import static org.springframework.http.HttpHeaders.CONTENT_TYPE;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+import com.github.tomakehurst.wiremock.client.WireMock;
+import com.github.tomakehurst.wiremock.junit.WireMockClassRule;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.io.Resources;
+import feign.FeignException;
+import java.io.IOException;
+import java.net.URL;
+import java.util.Map;
+import org.apache.commons.codec.Charsets;
+import org.assertj.core.util.Strings;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.cloud.contract.wiremock.WireMockSpring;
+import org.springframework.http.MediaType;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT,
+    properties = {
+        "core_case_data.api.url=http://localhost:4000"
+    })
+@ContextConfiguration
+@DirtiesContext
+public class CaseDataHelperTest {
+
+    private static final String AUTHORIZATION_HEADER_KEY = "Authorization";
+
+    private static final String SERVICE_AUTHORIZATION_HEADER_KEY = "ServiceAuthorization";
+
+    private static final String USER_AUTH_TOKEN = "Bearer eyJhbGciOiJIUzI1NiJ9.eyJqdGkiOiJhOGdyMjR2NmtiYXRibXFlcWthM3VuamVicSIsInN1YiI6I"
```

Comment:
Is there not a better way of doing this so we don't need to hardcode tokens into our tests?